docs/content/preview/explore/query-1-performance/pg-stat-progress-create-index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: View CREATE INDEX status with pg_stat_progress_create_index
+linkTitle: View CREATE INDEX status
+description: Use pg_stat_progress_create_index to get the CREATE INDEX command status, including the status of an ongoing concurrent index backfill, and the index build's progress reports.
+headerTitle: View CREATE INDEX status with pg_stat_progress_create_index
+image: /images/section_icons/index/develop.png
+menu:
+  preview:
+    identifier: pg-stat-progress-create-index
+    parent: query-tuning
+    weight: 450
+type: docs
+---
+
+You can add a new index to an existing table using the YSQL [CREATE INDEX](../../../api/ysql/the-sql-language/statements/ddl_create_index/#semantics) statement. YugabyteDB supports [online index backfill](https://github.com/yugabyte/yugabyte-db/blob/master/architecture/design/online-index-backfill.md), which is enabled by default. Using this feature, you can build indexes on non-empty tables online, without failing other concurrent writes. YugabyteDB also supports the [CREATE INDEX NONCONCURRENTLY](../../../api/ysql/the-sql-language/statements/ddl_create_index/#nonconcurrently) statement to disable online index backfill.
+
+### pg_stat_progress_create_index
+
+YugabyteDB supports the PostgreSQL `pg_stat_progress_create_index` view to report the progress of the CREATE INDEX command execution. Whenever CREATE INDEX is running, the `pg_stat_progress_create_index` view contains one row for each client connection that is currently running a CREATE INDEX command, and the row entry is cleared after the completion of command execution.
+
+The `pg_stat_progress_create_index` view can provide the following details:
+
+- Number of rows processed during an index backfill.
+- The current phase of the command with `initializing` or `backfilling` as the possible phases.
+- Index progress report for all the different configurations of an index or index build such as non-concurrent index builds, GIN indexes, partial indexes, and include indexes.
+
+The following table describes the view columns:
+
+| Column | Type | Description |
+| :----- | :--- | :---------- |
+| pid | integer | Process ID of backend that is running the CREATE INDEX. |
+| datid | OID | Object ID of the database to which this backend is connected. |
+| datname | name | Name of the database to which this backend is connected. |
+| relid | OID | Object ID of the indexed relation.|
+| index_relid | OID | Object ID of the index. |
+| command | text | The command that is running CREATE INDEX CONCURRENTLY, or CREATE INDEX NONCONCURRENTLY. |
```

Comment:
nit- code blocks around `CREATE INDEX CONCURRENTLY` and `CREATE INDEX NONCONCURRENTLY`